packages/pie-inline-choice/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "pie-inline-choice",
```

Comment:
name should be `@pie-elements/inline-choice`, any elements within this repo are scoped to `@pie-elements` any libs can be scoped to `@pie-libs`

Reply:
Sure will remember for future. Done